app/channels/iteration_channel.rb

```diff
@@ -0,0 +1,10 @@
+class IterationChannel < ApplicationCable::Channel
+  def subscribed
+    iteration = current_user.iterations.find(params[:iteration_id])
+    stream_for iteration
```

Comment:
One channel per iteration. This will probably get 1-1'd to the React component, so `IterationSummary`

Reply:
They're pretty cheap, but you could consider per student+exercise slug 🤷‍♂️ . Saves you from reconnecting per iteration, then again, this works fine :)

Reply:
Yeah. So I started there with an `IterationsList` component and `IterationsChannel`. I'm sort of 50/50 on it. My plan was still to have an `IterationsList` component+channel for if new iterations get submitted. But I guess I could hoist the channel so it's shared between the list, and just put logic in the `data`.

Reply:
Changed to a solution-level component as per this.